### DISCUSSION TITLE
Setup periodic discussion topic digest notifications

### DISCUSSION THREAD

**Discussion by cobra18t** [missionpinball/missionpinball#8] [General]
How do I configure github notifications to send me a summary of the activity on this discussion similar to what we have with Google Groups?

**Comment by jabdoa2** [comment]
I want that too!

**Comment by mattkemp** [comment]
I want it too! Unfortunately it doesn't look like it's a feature yet =( https://github.com/community/community/discussions/4075?sort=top

I get a copy of the new posts to my email and read them every day to see what people are doing and how they solve problems. If I can't get something similar set up, I'll probably lose touch with the community as I'm (let's be honest) probably too lazy to open up the website every day and read through the new posts and answers.

EDIT: it looks like they've created a webhook for Slack integration https://github.com/integrations/slack/issues/1026 but it's not comments, just new discussions.

This repo uses Issues for discussions for now, with the idea that issues can be turned into discussions: https://github.com/Ackee-Blockchain/woke/issues/62

**Reply by jabdoa2** [reply]
Looks like unwatching the repo is the only way to stop it. :-/
